conda-store-server/conda_store_server/server/auth.py

```diff
@@ -58,6 +70,74 @@ def authenticate(self, token):
 
 
 class RBACAuthorizationBackend(LoggingConfigurable):
+    role_mappings_version = Integer(
+        1,
+        help="Role mappings version to use: 1 (default, legacy), 2 (new, recommended)",
+        config=True,
+    )
```

Comment:
I make 1 the default so that users who already have existing role mappings could use conda-store without updating the config. Otherwise, their authorization would be broken until they either (1) set the value to 1 in the config or (2) inserted new roles into v2 tables using the new APIs.

Reply:
This is documented in `docs/administration.md`.

Reply:
What does this mean? Does this mean that new installs will use (1)?

Reply:
It's the same syntax as everywhere else: 1 is the default. What it means in practice: old installs will work without any changes. They'll use the existing (old) table.